fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTransactionType.java

```diff
@@ -63,8 +63,7 @@ public enum LoanTransactionType {
     DOWN_PAYMENT(28, "loanTransactionType.downPayment"), //
     REAGE(29, "loanTransactionType.reAge"), //
     REAMORTIZE(30, "loanTransactionType.reAmortize"), //
-    INTEREST_PAYMENT_WAIVER(31, "loanTransactionType.interestPaymentWaiver"), //
-    ;
+    INTEREST_PAYMENT_WAIVER(31, "loanTransactionType.interestPaymentWaiver"); //
```

Comment:
This is unrelated change. Please remove it

Reply:
Done! Removed